chia/util/streamable.py

```diff
@@ -181,6 +181,16 @@ def parse_bool(f: BinaryIO) -> bool:
         raise ValueError("Bool byte must be 0 or 1")
 
 
+def parse_uint32(f: BinaryIO, byteorder: str = "big") -> uint32:
+    size_bytes = f.read(4)
+    assert size_bytes is not None and len(size_bytes) == 4  # Checks for EOF
+    return uint32(int.from_bytes(size_bytes, byteorder))
+
+
+def write_uint32(f: BinaryIO, value: uint32, byteorder: str = "big"):
+    f.write(value.to_bytes(4, byteorder))
```

Comment:
we should have unit tests for these two new functions as well. (see `test_streamable.py`)

Reply:
Good point! See 158903a